bitnami/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 2.0.0
+version: 3.0.0
```

Comment:
Do you think this change will break the backward compatibility? If not, as this is not an addition of a new feature (that will be a minor) we should bump the patch version.

Reply:
I think yes due to `spec.selector.matchLabels` is immutable field. I have just "googled" some cases:
- https://github.com/jetstack/cert-manager/issues/1927
- https://github.com/istio/istio/issues/20142

Reply:
Oh, I see.
Then it would be great if you can add a section like this one: https://github.com/bitnami/charts/tree/master/bitnami/dokuwiki#upgrading to the README.md indicating the changes.

Reply:
I think a small description of what/why is the new version and some steps the user should follow if they want to upgrade.